pkg/codegen/docs/templates/resource.tmpl

```diff
@@ -54,17 +54,26 @@
 {{ template "constructor_args" }}
 {{ htmlSafe "{{% /choosable %}}" }}
 
-#### Resource Arguments
+## Resource Properties
```

Comment:
Just for consistency with the other headings (and their on-page nav items), what do you all think about using the resource name also? 

```suggestion
## {{.Header.Title}} Resource Properties
```